README.md

```diff
@@ -3,6 +3,62 @@
   <img alt="Open Sauced" src="https://i.ibb.co/7jPXt0Z/logo1-92f1a87f.png" width="300px">
   <h1>🍕 Open Sauced Browser Extension(s) 🍕</h1>
 </div>
+<p align="center">
+  <img src="https://img.shields.io/github/languages/code-size/open-sauced/browser-extensions" alt="GitHub code size in bytes">
+  <img src="https://img.shields.io/github/commit-activity/w/open-sauced/browser-extensions" alt="GitHub commit activity">
+  <a href="https://github.com/open-sauced/browser-extensions/issues">
+    <img src="https://img.shields.io/github/issues/open-sauced/browser-extensions" alt="GitHub issues">
+  </a>
+  <a href="https://discord.gg/U2peSNf23P">
+    <img src="https://img.shields.io/discord/714698561081704529.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2" alt="Discord">
+  </a>
+  <a href="https://twitter.com/saucedopen">
+    <img src="https://img.shields.io/twitter/follow/saucedopen?label=Follow&style=social" alt="Twitter">
+  </a>
+</p>
+
+## Running the project locally
+
+In order to run the project we need the following software binaries installed on our development machines:
```

Comment:
Instead of "we", using "you" would be better, as we already do in other docs.

Reply:
Let me rephrase that.